cartridges/int_affirm_pipelines/cartridge/scripts/checkoutnow/applyDiscount.js

```diff
@@ -0,0 +1,26 @@
+/**
+*  Checks coupon code and adds coupon to basket if valid
+* 	@input CurrentRequest : dw.system.Request
+*   
+*   @output obj : Object
+*
+*/
+
+/**
+ * 
+ * @param {Object} pdict pdict values
+ * @returns {Object} pipelet
+ */
+function execute( pdict ) {
+
+    var helper = require('*/cartridge/scripts/affirm/checkoutNowHelper');
+
+    var request = pdict.CurrentRequest;
+
+    var result = helper.ApplyDiscount();
+    pdict.obj = result.body;
+
+    return PIPELET_NEXT;
```

Comment:
How is this defined?

Reply:
The way `PIPELET_NEXT` keyword works is that since this script will only get called in the context of pipeline execution flow, returning the keyword executes the next logical step, which in this case is a util function that passes the response obj as a json:
<img width="763" alt="Screen Shot 2022-01-26 at 10 01 26 PM" src="https://user-images.githubusercontent.com/9426310/151300884-81c4a454-d8a2-4b56-b270-cf63ba43924d.png">